recipes/mss/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set version = "1.1.0" %}
+
+
+package:
+  name: mss
+  version: {{ version }}
+
+source:
+    git_url: https://bitbucket.org/wxmetvis/mss.git
+    git_rev: 1.1.0
+
+build:
+    number: 0
+    script: python setup.py install --record record.txt
+    entry_points:
+      - mss = msui.mss_pyui:main
+
+requirements:
+  build:
+    - basemap ==1.0.7
+    - chameleon >=2.24
+    - curl ==7.26
+    - cycler ==0.10.0
+    - execnet ==1.2.0
```

Comment:
@ocefpaf 

So I think I found the difference.for basemap. On some place of my code I missed to capsule like

```
try:
    import mpl_toolkits.basemap.pyproj as pyproj
except ImportError:
    import pyproj
```
additional for same functionality I have had to add manually

`conda install -c conda-forge basemap-data-hires`

without that package it crashes. Maybe that could be added as a dependency? Because it seems not needed for 1.0.7.